sql/mob_skill_lists.sql

```diff
@@ -3040,7 +3039,14 @@ INSERT INTO `mob_skill_lists` VALUES ('Asanbosam',885,392); -- ultrasonics
 INSERT INTO `mob_skill_lists` VALUES ('Asanbosam',885,394); -- blood_drain
 INSERT INTO `mob_skill_lists` VALUES ('Asanbosam',885,1155); -- suibsonics
 INSERT INTO `mob_skill_lists` VALUES ('Asanbosam',885,1156); -- marrow_drain
--- 886 to 887: free
+INSERT INTO `mob_skill_lists` VALUES ('Colo-Colo',886,392); -- ultrasonics
+INSERT INTO `mob_skill_lists` VALUES ('Colo-Colo',886,394); -- blood_drain
+INSERT INTO `mob_skill_lists` VALUES ('Colo-Colo',886,1155); -- subsonics
+INSERT INTO `mob_skill_lists` VALUES ('Colo-Colo',886,1156); -- marrow_drain
+INSERT INTO `mob_skill_lists` VALUES ('Furies',887,395); -- jetstream
+INSERT INTO `mob_skill_lists` VALUES ('Furies',887,393); -- sonic_boom
+INSERT INTO `mob_skill_lists` VALUES ('Furies',887,1157); -- slipstream
+INSERT INTO `mob_skill_lists` VALUES ('Furies',887,1158); -- turbulence
```

Comment:
Nothing wrong here but I have an ask if you can go through and add these skills to the bats that should have them out in the world. Looks like bat trios in CoP zones should use these.

Reply:
Couldn't apply to the following bats as they share zones with RoZ:
Succubus_Bats
Greater_Gaylas
Nightmare_Bats

Reply:
You'd have to make a new listing in mob_pools and a new skill_list (or assign the skill list you set to the other bats you could assign it to). And plug in the new mob_pool ID in their mob_groups listing.

Reply:
This doesn't have be part of this PR though, can do it any other time or I can later.

Reply:
I'll make sure to get it done in this PR so we don't forget about  it later :)